### PR TITLE
Header done

### PR DESCRIPTION
**Pull request by Dhwani0712**
fragmentshtml
home-controller
indexgtml
localhost_header
styles css1
styles css2

### CODE REVIEW

**Review by Rexzellius — APPROVED** [review]
Has the necessary links.

**Review by korina0430 — CHANGES_REQUESTED** [review]
Did you mean to push all these .lock and .bin files? Please remove them and maybe add them to the git ignore file

File: build/reports/tests/test/classes/org.launhcode.healthynutrition.HealthyNutritionApplicationTests.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+</div>
+<div id="footer">
+<p>
+<div>
+<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
+<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
+</label>
+</div>Generated by 
+<a href="http://www.gradle.org">Gradle 6.8.3</a> at Mar 15, 2021, 8:31:43 PM</p>
+</div>
```
Comment (korina0430) [inline build/reports/tests/test/classes/org.launhcode.healthynutrition.HealthyNutritionApplicationTests.html:134]: what does this file do? :o

**Review by korina0430 — COMMENTED** [review]
I am confused about the html and css files in the tests folder

**Comment by Dhwani0712** [comment]
> I am confused about the html and css files in the tests folder

I am not sure why they are showing in test file, I checked and they are in resources folder in Intellij

**Pull request closed without merging**